src/terms.ml

```diff
@@ -59,6 +59,8 @@ type term =
   (** Module in which it is defined. *)
   ; sym_def   : term option ref
   (** Definition of the symbol. *)
+  ; sym_implicits : bool list
+  (** Implicitness of arguments for a function def *)
```

Comment:
Change the comment to `(** Implicitness of arguments. *)`.

Reply:
Done.